meson.build

```diff
@@ -206,6 +206,14 @@ foreach tuple : _customdirs
   customdir = get_option(dir_option)
   if customdir == ''
     customdir = def_path
+  else
+    if customdir.endswith('/')
+      customdir = customdir.substring(0, -1)
+    endif
```

Comment:
someone will always want to cause trouble by doing ` meson setup -D moduledir=/got/you///`

instead of the `if`, maybe just do: `customdir = customdir.strip(strip_chars="'')`

Reply:
Initially opted against `.strip()` since it removes both leading and trailing symbols.

Updated to use them and add the leading one... As a bonus, it will convert `///bar/baz////` to `/bar/baz`.

Reply:
humn.... and we don't have rstrip(). :-/